src/Command/ContentHubEntityScanOrphaned.php

```diff
@@ -0,0 +1,169 @@
+use Symfony\Component\Console\Helper\Table;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\EventDispatcher\EventDispatcherInterface;
+
+/**
+ * Provides command to scan for orphaned entities in Content Hub.
+ */
+class ContentHubEntityScanOrphaned extends Command implements PlatformCommandInterface {
+
+  use PlatformCommandTrait;
+  use PlatformCmdOutputFormatterTrait;
+
+  /**
+   * {@inheritdoc}
+   */
+  protected static $defaultName = 'ach:entity-scan:orphaned';
+
+  /**
+   * The platform command executioner.
+   *
+   * @var \Acquia\Console\Helpers\PlatformCommandExecutioner
+   */
+  protected $platformCommandExecutioner;
+
+  /**
+   * {@inheritdoc}
+   */
+  public static function getExpectedPlatformOptions(): array {
+    return ['source' => PlatformCommandInterface::ANY_PLATFORM];
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  protected function configure() {
+    $this->setDescription('Scan for orphaned entities in the subscription.');
+    $this->addOption('delete', 'd', InputOption::VALUE_NONE, 'Delete found entities');
+    $this->addOption('json', '', InputOption::VALUE_NONE, 'Return output as json');
+    $this->addOption('with-db', '', InputOption::VALUE_NONE, 'Check if the orphaned entities are present in DB. If yes exclude them from the result.');
+    $this->addOption('with-interest-list', '', InputOption::VALUE_NONE, 'Check if the orphaned entities are present on the interest list. If yes exclude them from the result.');
+    $this->addOption('only-uuids', '', InputOption::VALUE_NONE, 'Returns only uuids in a non-formatted way, 1 by line. (Omits json flag)');
+    $this->setAliases(['ach-eso']);
+  }
+
+  /**
+   * ContentHubVersion constructor.
+   *
+   * @param \Symfony\Component\EventDispatcher\EventDispatcherInterface $dispatcher
+   *   The dispatcher service.
+   * @param \Acquia\Console\Helpers\PlatformCommandExecutioner $platformCommandExecutioner
+   *   The platform command executioner.
+   * @param string|null $name
+   *   The name of the command.
+   */
+  public function __construct(EventDispatcherInterface $dispatcher, PlatformCommandExecutioner $platformCommandExecutioner, $name = NULL) {
+    parent::__construct($name);
+    $this->platformCommandExecutioner = $platformCommandExecutioner;
+    $this->dispatcher = $dispatcher;
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  protected function execute(InputInterface $input, OutputInterface $output): int {
+    $drushOptions = [
+      '--drush_command' => 'acquia:contenthub:entity-scan:orphaned',
+      '--drush_args' => [
+        '--json',
+        '--with-db',
+        '--with-interest-list',
+      ],
+      '--json' => '',
+    ];
+
+    $raw = $this->platformCommandExecutioner->runWithMemoryOutput(DrushWrapper::$defaultName, $this->getPlatform('source'), $drushOptions);
+    $data = $this->extractAndAggregateJson($raw);
```

Comment:
I couldn't find this function. Could you please help me with that?

Reply:
https://github.com/acquia/contenthub-console-helpers/pull/15
I left out this one, my bad!